framework_tvb/tvb/interfaces/web/controllers/exceptions.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+#
+# TheVirtualBrain-Framework Package. This package holds all Data Management, and
+# Web-UI helpful to run brain-simulations. To use it, you also need do download
+# TheVirtualBrain-Scientific Package (for simulators). See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2020, Baycrest Centre for Geriatric Care ("Baycrest") and others
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this
+# program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+#
+#
+"""
+Exceptions for the web interface.
+"""
+
+from tvb.basic.exceptions import TVBException
+
+
+class InvalidFormValues(TVBException):
+    """
+    Exception to be thrown in case of existing
+    some invalid values in a form.
+    """
+
+    def __init__(self, message, error_dict=None):
+        TVBException.__init__(self, message)
+        self.error_dict = error_dict
+
+    def display_full_errors(self):
+        if self.error_dict:
+            result = {}
+            for name, item in self.error_dict.items():
+                result[name] = str(item)
+            return result
```

Comment:
We should display both `self.message` and the concatenated `error_dict.`
Maybe put every entry in the dict on a separate line, and show both the key and the value from the dictionary